src/components/SimpleInput.js

```diff
@@ -0,0 +1,55 @@
+const React = require('react');
+const Radium = require('radium');
+
+const StyleConstants = require('../constants/Style');
+
+const Input = React.createClass({
+  propTypes: {
+    name: React.PropTypes.string,
+    placeholder: React.PropTypes.string,
+    type: React.PropTypes.string,
+    valid: React.PropTypes.bool
+  },
+
+  getDefaultProps () {
+    return {
+      type: 'text',
+      valid: true
+    };
+  },
+
+  render () {
+    const styles = this.styles();
+
+    return (
+      <input
+        {...this.props}
+        name={this.props.name}
+        style={Object.assign({}, styles.wrapper, this.props.style)}
+        type={this.props.type}
```

Comment:
Curious about the spread operator here, which could also address my earlier question about value and defaultValue...

If those are passed in as props, would they need to be pulled out or would they be included with the `...this.props`?

If so, do you still need the name and type?